bottomless/src/replicator.rs

```diff
@@ -967,7 +967,7 @@ impl Replicator {
         &mut self,
         generation: Uuid,
         utc_time: Option<NaiveDateTime>,
-    ) -> Result<RestoreAction> {
```

Comment:
what does this bool mean? Please add a comment

Reply:
Ok, now I went over the whole change and I'm even more confused - why was this parameter added at all? It looks like it's not used anywhere, you just ignore it above.

Reply:
it is used there, I will had a docstring
https://github.com/libsql/sqld/pull/646/files/3b76a120da75aaf2794f4f50f039a4b35b4e6c8d#diff-3035393d5fdf2fb87106000360e07ea7386f5ef923d280dd6eea3dfc9181ba39R453